apps/swirl-docs/src/pages/api/tokens/v0/utils.ts

```diff
@@ -0,0 +1,74 @@
+import { SWIRL_TOKENS_DIST_PATH } from "@swirl/lib/navigation";
+import fs from "fs";
+import path from "path";
+
+export const OVERVIEW_HTML = fs.readFileSync(
```

Comment:
**thought:** All readFile operation could be wrapped in a function, so the files get loaded into memory on demand. Tradeoff would be slightly longer response times.